src/web/BL_Python/web/application.py

```diff
@@ -138,15 +180,15 @@ def configure_openapi(config: Config, name: Optional[str] = None):
             "OpenAPI configuration is empty. Review the `openapi` section of your application's `config.toml`."
         )
 
-    # host configuration set up
-    # TODO host/port setup should move into application initialization
-    # and not be tied to connexion configuration
-    host = "127.0.0.1"
-    port = 5000
-    # TODO replace SERVER_NAME with host/port in config
-    if environ.get("SERVER_NAME") is not None:
-        (host, port_str) = environ["SERVER_NAME"].split(":")
-        port = int(port_str)
+    ## host configuration set up
+    ## TODO host/port setup should move into application initialization
+    ## and not be tied to connexion configuration
+    # host = "127.0.0.1"
+    # port = 5000
+    ## TODO replace SERVER_NAME with host/port in config
+    # if environ.get("SERVER_NAME") is not None:
+    #    (host, port_str) = environ["SERVER_NAME"].split(":")
+    #    port = int(port_str)
```

Comment:
I found a thing! Please remove the commented-out code.

Reply:
Regarding the commented code, most of these blocks are aspects that I need to refactor into the OpenAPI middlewares, but can be figured out at a later date. Not exactly dead code, and definitely not code I want to forget exists. Would you accept if I refactored these blocks of code into roughly where they'll end up, with ample TODOs and FIXMEs?

Reply:
Absolutely! After this first one I tried to leave alone anything with a matching `TODO` or `FIXME` and just focus on contextless blocks.